backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java

```diff
@@ -433,33 +448,53 @@ protected void updateSafeInsets() {
 		safeInsetLeft = 0;
 		safeInsetRight = 0;
 		safeInsetBottom = 0;
+		safeInsetRawTop = 0;

```

Comment:
The new raw values are the same as the other ones, multiplied with a (more or less) constant... would it be better to just have the four logical values and multiply when getter method is called?

Reply:
I thought to leave these "cached" as it's a slight performance hit doing a float multiplication if this is called in every `render()`. Acknowledge it's negligible so happy to change if you prefer.

Reply:
Background of the save fields is less the performance hit but more that otherwise we would have the need to call the native methods 4 times to get the values. I don't think there is any performance hit here if the values are multiplied, so I would prefer the less polluted graphics class. But that's my opinion, let's see if others have an opinion on this.

Reply:
Is there a good reason to not just reuse the existing fields to store pixel offsets instead of point offsets? I don't see any just by looking at the file in GitHub and it would solve this whole discussion.

Reply:
You are completely right, they can be reused.

Reply:
Yep, and even better, we could have a single int[] to hold the 4 values.

Reply:
Yep, and even better, we could have a single int[] to hold the 4 values.